src/adapters/gumgum.js

```diff
@@ -10,8 +10,16 @@ const GumgumAdapter = function GumgumAdapter() {
 
   const bidEndpoint = `https://g2.gumgum.com/hbid/imp`;
 
-  const WINDOW = global.top;
-  const SCREEN = WINDOW.screen;
+  let WINDOW;
+  let SCREEN;
+
+  try {
+    WINDOW = global.top;
+    SCREEN = WINDOW.screen;
+  } catch (error) {
+    utils.logError(error);
+    return;
```

Comment:
We do need to access `window.top`, since the service was built with the idea of having the top being the host of the script.

Especially with the In-Screen service which uses the placement a 1x1 pixel, because our In-Screen uses fluid designs across the window width. If we do not have access to `top`, the In-Screen should not run.

@mkendall07 I can work on this if you like.